in_toto/gpg/functions.py

```diff
@@ -0,0 +1,168 @@
+"""
+<Module Name>
+  gpg/functions.py
+
+<Author>
+  Santiago Torres-Arias <santiago@nyu.edu>
+
+<Started>
+  Nov 15, 2017
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  publicly-usable functions for exporting public-keys, signing data and
+  verifying signatures.
+"""
+import subprocess
+import shlex
+
+import in_toto.log
+from in_toto.gpg.constants import (GPG_EXPORT_PUBKEY_COMMAND, GPG_SIGN_COMMAND,
+    SIGNATURE_HANDLERS)
+from in_toto.gpg.common import parse_signature_packet, parse_pubkey_packet
+
+import securesystemslib.formats
+
+
+def gpg_sign_object(content, keyid=None, homedir=None):
+  """
+  <Purpose>
+    Calls gpg command line utility to sign the passed content with the key
```

Comment:
Should you clarify that `gpg2` is called?  Consider pasting the full GPG command, or stating that GPG_SIGN_COMMAND is called.